vignettes/articles/zip_zap_models.Rmd

```diff
@@ -123,17 +136,23 @@ $$
 where $p=\text{logit}^{-1}(\theta)$
 
 ```{r fig-pred-zip, fig.cap="Predictions from zero-inflated model",fig.width=7, fig.height=5, out.width="80%", fig.align='center'}
-fit_zip <- bru(comps,
-                like(family = "zeroinflatedpoisson1", data=counts_df,
-                     formula = count~vegetation+elevation+field+Intercept,
-                     E=area,
-                     domain = list(geometry = px_mesh)))
+fit_zip <- bru(
+  comps,
+  like(
+    family = "zeroinflatedpoisson1", data = counts_df,
+    formula = count ~ vegetation + elevation + field + Intercept,
+    E = area,
+    domain = list(geometry = px_mesh)
```

Comment:
Since this is not a "cp" model, the domain argument will have no effect, and should be removed.